code/modules/mob/living/carbon/carbon.dm

```diff
@@ -106,6 +106,8 @@
 			Weaken(3)
 	if(override)
 		return override
+	else if(shock_damage > 400)

```

Comment:
this is actually really not good because non-human carbon mobs (aliums) have a different health pool